Xamarin.Forms.Core/Items/ItemsView.cs

```diff
@@ -0,0 +1,142 @@
+﻿using System;
+using System.Collections;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Text.RegularExpressions;
+using Xamarin.Forms.Platform;
+
+namespace Xamarin.Forms
+{
+	// TODO hartez 2018/06/23 13:42:22 Trying this out for a nicer read in OnElementPropertyChanged, not sure if I like it yet	
+	public static class PropertyChangedEventArgsExtensions
+	{
+		public static bool Is(this PropertyChangedEventArgs args, BindableProperty property)
+		{
+			CoreFlags.VerifyCollectionViewFlagEnabled();
+			return args.PropertyName == property.PropertyName;
+		}
+
+		public static bool IsOneOf(this PropertyChangedEventArgs args, params BindableProperty[] properties)
+		{
+			CoreFlags.VerifyCollectionViewFlagEnabled();
+			for (int n = 0; n < properties.Length; n++)
```

Comment:
properties is an array, `foreach` is free

Reply:
Fixed.